db/revtree_test.go

```diff
@@ -265,7 +268,7 @@ func TestRevTreeAddRevision(t *testing.T) {
 	tempmap := testmap.copy()
 	goassert.DeepEquals(t, tempmap, testmap)
 
-	tempmap.addRevision("testdoc", RevInfo{ID: "4-four", Parent: "3-three"})
+	_ = tempmap.addRevision("testdoc", RevInfo{ID: "4-four", Parent: "3-three"})
```

Comment:
Still a lot of test errors that are not being checked. I expect all of these are expected to have no error, so we should check them to be safe.

I think the only places we should ignore test errors are inside benchmarks where we don't want the overhead of assertions.